src/scenes/Home/index.js

```diff
@@ -24,16 +26,27 @@ class Home extends Component {
     super(props)
 
     this.state = {
-      searchText: ''
+      searchText: '',
+      nodes: null
     }
   }
 
   handleSearchChange (value) {
     this.setState({ searchText: value })
+
+    if (value !== '') {
+      const onSearchSuccess = nodes => {
+        this.setState({ nodes })
+      }
+
+      search.simpleQuery(value, onSearchSuccess.bind(this))
```

Comment:
Je trouve que `nodes => onSearchSuccess(nodes)` serait plus explicite que  `onSearchSuccess.bind(this)`